cmd/podman/images/build.go

```diff
@@ -229,7 +229,11 @@ func extractContextAndFiles(args, files []string) (string, []string, error) {
 		if utils.FileExists(filepath.Join(contextDir, "Containerfile")) {
 			containerFiles = append(containerFiles, filepath.Join(contextDir, "Containerfile"))
 		} else {
-			containerFiles = append(containerFiles, filepath.Join(contextDir, "Dockerfile"))
+			tmpDir := contextDir
```

Comment:
We want to leave in the Dockerfile appending as it was.  We want the build to default to using a Containerfile if present, if not, then to using a Dockerfile if present.   Perhaps add this above the else:

```
else if utils.FileExists(filepath.Join(contextDir, "Dockerrfile")) {
    containerFiles = append(containerFiles, filepath.Join(contextDir, "Dockerfile"))
} else {
```